onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -667,10 +668,10 @@ contract CartesiDAppTest is TestBase {
         for (uint256 i; i < outputEnums.length; ++i) {
             LibServerManager.OutputEnum outputEnum = outputEnums[i];
             if (outputEnum == LibServerManager.OutputEnum.VOUCHER) {
-                Voucher memory voucher = getVoucher(i);
+                Voucher memory voucher = getVoucher(OutputName(i));
```

Comment:
This part got a little weird. Maybe we should keep `getNotice` and `getVoucher` the way they were and create new `getNoticeByName` and `getVoucherByName` that do this conversion from enumeration to index and call internally `getNotice` and `getVoucher` respectively.

Reply:
What I find it weird is that, in this function, we are converting `uint256` into `OutputName`, and then in `getVoucher`, we are converting it back into `uint256`.

Reply:
Agreed and addressed :)